tests/test_core.py

```diff
@@ -5312,6 +5312,9 @@ def test_fs_llseek(self):
   def test_fs_64bit(self):
     self.do_runf(path_from_root('tests', 'fs', 'test_64bit.c'), 'success')
 
+  def test_sigalrm(self):
+    self.do_runf(path_from_root('tests', 'sigalrm.cpp'), '')
```

Comment:
It seems a little excessive to run this in all optimization levels, unless I'm missing a possible interaction? `other` makes more sense otherwise.

Reply:
But all the other tests for stdlib functions are currently here in core:

test_unistd_access, test_uname, etc, etc..

Reply:
Maybe there is a larger question about which suite all these tests should go in?

Reply:
I think some stdlib things may make sense to run in core, as they may be influenced by optimizations, etc. I'm not sure this particular function does? If we don't think there's a reason, better to speed up CI by putting it in `other` I think.